### ISSUE TITLE
My minecraft crashes when I join a specific server

### ISSUE THREAD

**Issue by ILikeTrains27** [ImpactDevelopment/ImpactClient#428]
### What happens?

My client crashes
### When does this happen?

A while after I join I join the server metoothanks.us (Except maybe this is just because I play this server the most.) and then once it happens the first time, instantly when I log in. I can fix it temporarily by logging into the server without impact, and then switching back to impact.
### Exact steps to reproduce?
You probably don't want to try to reproduce it because it takes a while and probably won't work but whatever
- Log on to metoothanks.us and have all of my mods enabled
- Wait a while and do some stuff like break blocks and jump.

### Game output?

```
I think the game output is a log so here's on of mine https://pastebin.com/8rVjcGrw
```

### Crash report?

```
https://pastebin.com/kHDMgmNE
```

- Impact Version: 4.1 for 1.12.2,
- Using Optifine? No
- Enabled mods: 
Middle Click Friends
Levitation Control
Auto Reconnect
Anti Overlay
ItemPhysics
Auto Totem
Auto Armor
CameraClip
StorageESP
Auto Cheat
Nametags
Anti Blind
Inventory
Respawn
Auto Eat
No Slow
Weather
Criticals
Tracers
Light
- Operating System:  Windows 10 64bit

**Comment by Thunder33345** [comment]
did you died ingame with autorespawn crashing it?
it might be related to dying and autorespawn fails to respawn thus crashing
THEN crashing again when joining and meet the respawn screen
this is a theory as i dont really have access to the code to analyze the obfuscated stack

**Comment by ILikeTrains27** [comment]
No, I didn't die

**Comment by ZeroMemes** [comment]
Turn the ``Random`` value on ``Auto Eat`` to a value greater than 0, I've fixed this for 4.2
